api_fhir/converters/claimResponseConverter.py

```diff
@@ -113,10 +122,19 @@ def get_imis_claim_item_by_code(cls, code, imis_claim_id):
         result = ClaimItem.objects.filter(item_id__in=Subquery(item_code_qs.values('id')), claim_id=imis_claim_id)
         return result[0] if len(result) > 0 else None
 
+    @classmethod
+    def build_fhir_claim_add_item(cls, fhir_claim_response, claim_item):
+        add_item = ClaimResponseAddItem()
+        item_code = claim_item.service.text
+        add_item.sequenceLinkId.append(claim_item.sequence)
+        add_item.service = cls.build_codeable_concept(code=item_code)
```

Comment:
So you decide to use `Group, Service or Product` to store the information about the item code and this should increase the performance?